include/dts-functions.sh

```diff
@@ -1730,42 +1623,52 @@ send_dts_logs(){
 }
 
 check_if_fused() {
-  file_path="/sys/class/mei/mei0/fw_status"
-
-  if [[ ! -f $file_path ]]; then
-    echo "File not found: $file_path"
+  local _file_path
+  _file_path="/sys/class/mei/mei0/fw_status"
+  local _file_content
+  local _hfsts6_value
+  local _line_number
+  local _hfsts6_binary
+  local _binary_length
+  local _padding
+  local _zeros
+  local _bit_30_value
+
+  if ! $FSREAD_TOOL test -f "$_file_path"; then
+    print_error "File not found: $_file_path"
     return 2
   fi
 
-  hfsts6_value=""
-  line_number=1
+  _file_content="$($FSREAD_TOOL cat $_file_path)"
+
+  _fsts6_value=""
+  _line_number=1
   while IFS= read -r line; do
-    if [[ $line_number -eq 6 ]]; then
-      hfsts6_value=$line
+    if [[ $_line_number -eq 6 ]]; then
+      _hfsts6_value="$line"
```

Comment:
@PLangowski, you are checking for a value of a specific register here, am I right? If so, is the layout of these registers is the same for all platforms or MEI's? Could you point me out to the documentation where this register is described? I want to add some comments here, because the purpose and the method of this functions is not clear.

Reply:
The layout of the register is the same for all platforms, but the problem is that not all platforms have this register.  We are currently working on this problem https://github.com/Dasharo/dts-scripts/pull/37. As for the documentation, I don't know if there is any.

Reply:
Will be addressed here https://github.com/Dasharo/dts-scripts/pull/37#pullrequestreview-2381558744.